_includes/views-vues/assurance-level-requirement/assurance-level-resultats.html

```diff
@@ -0,0 +1,215 @@
+          { "type": "conditional", "inputs": [
+            { "type": "=", "inputs": [
+              { "type": "number", "query": "#max-assurance-level-used" },
+              {{ level }}
+            ] }
+        ], "actionsTrue": [
+          { "type": "removeClass", "outputTarget": "#assurance-{{ level }}-table", "class": "hidden" }
+        ], "actionsFalse": [
+          { "type": "addClass", "outputTarget": "#assurance-{{ level }}-table", "class": "hidden" }
+        ] }
+      ] }'>
+     {{ page.assurance[ assuranceLevel ] }}
+    </div>{%
+  endfor %}
+  <div id="assurance-NA-table" class="hidden"></div>
+</td>
+</tr>
+<tr>
+  <td colspan="5" style="text-align: center"> {{ page.version }} </td>
+</tr>
+</tbody>
+</table>
+</div>
+
+<button type="button" class="btn btn-default" onclick="javascript:PrintElem($( '#table-results-div' ), 'LOA Assessment - ' + $('#table-detail-3').text() + ' - ' + $('#table-detail-2').text());">Print / Download PDF</button>
+<button type="button" class="btn btn-default wb-format-gen" data-wb-format-gen='{ "type": "csv", "rowSelector": "tr:not(.hidden)", "colSelector": "th, td div:not(.hidden), .non-scoring-result td", "container": "#table-results-div", "filename": "assurance-level-assessment" }'>{{ site.DownloadCSV[page.lang] }}</button>
+
+<script language="javascript" type="text/javascript">
+  function PrintElem(elem, title, offset)
+{
+    // Title constructor
+    title = title || $('title').text();
+    // Offset for the print
+    offset = offset || 0;
+
+    // Loading start
+    var dStart = Math.round(new Date().getTime()/1000),
+        $html = $('html');
+        i = 0;
+
+    // Start building HTML
+    var HTML = '<html';
+
+    if(typeof ($html.attr('lang')) !== 'undefined') {
+        HTML+=' lang=' + $html.attr('lang');
+    }
+
+    if(typeof ($html.attr('id')) !== 'undefined') {
+        HTML+=' id=' + $html.attr('id');
+    }
+
+    if(typeof ($html.attr('xmlns')) !== 'undefined') {
+        HTML+=' xmlns=' + $html.attr('xmlns');
+    }
+
+    // Close HTML and start build HEAD
+    HTML+='><head>';
+
+    // Get all meta tags
+    $('head > meta').each(function(){
+        var $this = $(this),
+            $meta = '<meta';
+
+        if(typeof ($this.attr('charset')) !== 'undefined') {
+            $meta+=' charset=' + $this.attr('charset');
+        }
+        if(typeof ($this.attr('name')) !== 'undefined') {
+            $meta+=' name=' + $this.attr('name');
+        }
+        if(typeof ($this.attr('http-equiv')) !== 'undefined') {
+            $meta+=' http-equiv=' + $this.attr('http-equiv');
+        }
+        if(typeof ($this.attr('content')) !== 'undefined') {
+            $meta+=' content=' + $this.attr('content');
+        }
+
+        $meta+=' />';
+        HTML+= $meta;
+        i++;
+    }).promise().done(function(){
```

Comment:
This won't run on IE11, I've added ESLint to the build now, and I'm guessing it will flag this

Reply:
What will the issue be? (I'm not familiar with IE)
I did have the print function work fine when I tested it on IE right now.

Reply:
IE 11 (not Edge) doesn't support promises at all https://caniuse.com/promises
It's possible someone added a polyfil for it already, but I didn't see it in my quick skim

Reply:
There must be, since I have it working on IE.
Here's the function inside the promise().done, with the relevant Meta and CSS in the HTML variable:
![IE11](https://user-images.githubusercontent.com/55841241/93802917-14651080-fc12-11ea-8a8b-79683eb26fbe.PNG)